src/EA.Weee.Web/Areas/AATFReturn/ViewModels/SelectReportOptionsViewModel.cs

```diff
@@ -6,7 +6,10 @@
     using EA.Prsd.Core;
     using EA.Weee.Core.AatfReturn;
     using EA.Weee.Core.DataReturns;
+    using FluentValidation.Attributes;
+    using Validation;
 
+    [Validator(typeof(SelectReportOptionsViewModelValidator))]
```

Comment:
Can a unit test be added to ensure the model has this attribute with the SelectReportOptionsViewModelValidator

Reply:
Added